src/Cell.js

```diff
@@ -93,7 +99,7 @@ export class Cell<Data: Types.CellBase, Value> extends PureComponent<
     if (selected && this.root) {
       setCellDimensions({ row, column }, getOffsetRect(this.root));
     }
-    if (this.root && active && mode === "view") {
+    if (this.root && this.isTableCell() && active && mode === "view") {
```

Comment:
Did you understand the purpose of this block? I disabled it entirely and didn’t see any drawbacks.

Reply:
This block is needed, without the `this.root.focus()` you will loose the interaction through keyboard with the spreadsheet. For example, using the navigation arrows  to select nearby cells.